test/integration/test-custom-filename.js

```diff
@@ -26,6 +30,17 @@ var server = http.createServer(function(req, res) {
 
   var form = new IncomingForm({uploadDir: common.dir.tmp});
 
+  // Data(`filename` field) after form.parse() automatically decodes some characters.
+  // Therefore, check `req` to see if filename is properly escaped.
+  // https://html.spec.whatwg.org/#multipart-form-data
+  var body = '';
+  req.on('data', function (chunk) {
+    body += chunk;
+  });
+  req.on('end', function () {
+    assert(body.includes(escapedContentDispositionHeader), 'Expects encode "filename" field');
```

Comment:
```suggestion
    assert(body.indexOf(escapedContentDispositionHeader) >= 0, 'Expects encode "filename" field');
```
similarly, includes isn't available in everything either.